From f63c04752f8692e317c5f0b294121f2b7287399e Mon Sep 17 00:00:00 2001
From: Ahmed ARIF <arif.ing@outlook.com>
Date: Sun, 21 Sep 2025 16:35:57 +0200
Subject: [PATCH] Apply 0022-aarch64-Add-stdcall-and-cdecl-attributes

---
 gcc/config/aarch64/aarch64.cc | 2 ++
 1 file changed, 2 insertions(+)

Index: gcc-16.1.0/gcc/config/aarch64/aarch64.cc
===================================================================
--- gcc-16.1.0.orig/gcc/config/aarch64/aarch64.cc
+++ gcc-16.1.0/gcc/config/aarch64/aarch64.cc
@@ -928,6 +928,8 @@ static const attribute_spec aarch64_gnu_
   { "Advanced SIMD type", 1, 1, false, true,  false, true,  NULL, NULL },
   { "SVE type",		  3, 3, false, true,  false, true,  NULL, NULL },
   { "SVE sizeless type",  0, 0, false, true,  false, true,  NULL, NULL },
+  { "cdecl",		  0, 0, false, false, false, false, NULL, NULL },
+  { "stdcall",	  0, 0, false, false, false, false, NULL, NULL },
 #if TARGET_DLLIMPORT_DECL_ATTRIBUTES
   { "dllimport", 0, 0, false, false, false, false, handle_dll_attribute, NULL },
   { "dllexport", 0, 0, false, false, false, false, handle_dll_attribute, NULL },
